### PR TITLE
[DPE-3721] - chore: use tools-log4j.properties for run_bin_command

### PR DESCRIPTION
**Pull request by marcoppenheimer**
#### `chore: pass tools-log4j.properties to run_bin_command`
- Is related to https://github.com/canonical/charmed-kafka-snap/pull/38
- Removes the overly verbose logging defaults when SSH'd to a machine and manually running bin commands during debugging
- Keeps the verbose logging for charm runtime error introspection
#### `refactor: minor tidy-ups`
- Renaming `KafkaConfigManager` --> `ConfigManager`. The 'Kafka' part seemed redundant
- JAAS indentation was wonky with multiline strings, now it's aligned in a sensible way

### CODE REVIEW

File: src/managers/config.py [hunk 1]
```diff
@@ -415,14 +427,15 @@ def zk_jaas_config(self) -> str:
...
             String of Jaas config for ZooKeeper auth
         """
-        return f"""
-Client {{
-    org.apache.zookeeper.server.auth.DigestLoginModule required
-    username="{self.state.zookeeper.username}"
-    password="{self.state.zookeeper.password}";
-}};
-
+        return inspect.cleandoc(
```
Comment (deusebio) [inline src/managers/config.py:430]: **question** what is this for?

**Review by deusebio — APPROVED** [review]
LGTM!

File: src/managers/config.py [hunk 2]
```diff
@@ -415,14 +427,15 @@ def zk_jaas_config(self) -> str:
...
+            f"""
+            Client {{
+                org.apache.zookeeper.server.auth.DigestLoginModule required
+                username="{self.state.zookeeper.username}"
+                password="{self.state.zookeeper.password}";
+            }};
         """
+        )
```
Comment (Batalex) [inline src/managers/config.py:438]: praise: TIL

Usually I would use [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) but it looks like a better option

**Review by Batalex — APPROVED** [review]
LGTM.
We could have kept `KafkaConfigManager`, since we might have a `CruiseControlManager` coming soon (and maybe a kraft one if the config differs enough), but this won't change much comprehension-wise

**Review by zmraul — APPROVED** [review]
LGTM. I agree with Alex though, config managers will probably get reverted/changed soon.

File: src/managers/config.py (hunk 1 above)
Comment (marcoppenheimer) [reply src/managers/config.py:432]: Aesthetics. Having it out of alignment was frustrating.

**Pull request merged**